tests/sparsezoo/git_server_model/test_models.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+pytest /Users/george/neuralmagic/sparsezoo/tests/sparsezoo/git_server_model/test_models.py
+
+
+"""
+from sparsezoo.git_server_model.models.model_metadata import ModelMetadata
+from sparsezoo.git_server_model.utils import (
+    extract_git_server_metadata,
+    local_load,
+    web_load,
+)
+
+
+def test_model_card_local_load():
+    """
+    Test local_load()
+    """
+    # Your modelcard path
+    path = "/Users/george/Desktop/model.md"
```

Comment:
tests will not always have access to this directory :)
I suggest you mock the input.